### PR TITLE
JDK-8278346: java/nio/file/Files/probeContentType/Basic.java fails on Linux SLES15 machine

### PR DESCRIPTION
**Pull request by MBaesken**
Hello, please review this small test adjustment.

The test java/nio/file/Files/probeContentType/Basic.java fails on a SLES15 Linux machine with this output :
Content type: application/x-bzip; expected: [application/bz2, application/x-bzip2]
Content type: application/x-genesis-rom; expected: [text/markdown]
Content type: application/x-php; expected: [text/plain, text/php]
Content type: application/x-rar; expected: [application/rar, application/vnd.rar]
java.lang.RuntimeException: Test failed!
        at Basic.main(Basic.java:192)
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
        at java.base/java.lang.reflect.Method.invoke(Method.java:577)
        at com.sun.javatest.regtest.agent.MainWrapper$MainThread.run(MainWrapper.java:127)
        at java.base/java.lang.Thread.run(Thread.java:833)

For see the strange 'application/x-genesis-rom' mapping see also
markdown files ending in .md get marked as 'application/x-genesis-rom'
https://bugs.launchpad.net/ubuntu/+source/shared-mime-info/+bug/611763

The other mappings look okay to me but sems we need to add them to the test.

Best regards, Matthias

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8278346](https://bugs.openjdk.java.net/browse/JDK-8278346): java/nio/file/Files/probeContentType/Basic.java fails on Linux SLES15 machine

### Reviewers
 * [Aleksey Shipilev](https://openjdk.java.net/census#shade) (@shipilev - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/6761/head:pull/6761` \
`$ git checkout pull/6761`

Update a local copy of the PR: \
`$ git checkout pull/6761` \
`$ git pull https://git.openjdk.java.net/jdk pull/6761/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 6761`

View PR using the GUI difftool: \
`$ git pr show -t 6761`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/6761.diff">https://git.openjdk.java.net/jdk/pull/6761.diff</a>

</details>

### CODE REVIEW

File: test/jdk/java/nio/file/Files/probeContentType/Basic.java [hunk 1]
```diff
@@ -110,8 +110,9 @@ static int checkContentTypes(ExType[] exTypes)
                             + " cannot be determined");
                     failures++;
                 } else if (!expectedTypes.contains(type)) {
-                    System.err.printf("Content type: %s; expected: %s%n",
-                        type, expectedTypes);
+                    System.err.printf("For extension " + extension
+                            + " we got content type: %s; expected: %s%n",
+                            type, expectedTypes);
```
Comment (FrauBoes) [inline test/jdk/java/nio/file/Files/probeContentType/Basic.java:115]: For consistency, we can pass `extension` as argument, too:
```
System.err.printf("For extension %s we got content type: %s; expected: %s%n", 
        extension, type, expectedTypes);
```

**Comment by FrauBoes** [comment]
Apart from the small comment LGTM (I'm not a reviewer)

File: test/jdk/java/nio/file/Files/probeContentType/Basic.java (hunk 1 above)
Comment (MBaesken) [reply test/jdk/java/nio/file/Files/probeContentType/Basic.java]: Thanks, I adjusted the output.

**Review by shipilev — APPROVED** [review]

**Pull request closed without merging**